proto/waves/node/grpc/accounts_api.proto

```diff
@@ -57,3 +57,12 @@ message ScriptData {
     string script_text = 2;
     int64 complexity = 3;
 }
+
+message LeaseResponse {
+    bytes leaseId = 1;
+    bytes originTransactionId = 2;
+    bytes sender = 3;
+    bytes recipient = 4;
```

Comment:
In `LeaseTransactionData` `recipient` has the `Recipient` type, but here it's replaced by stripped account only. https://github.com/wavesplatform/protobuf-schemas/blob/ac1aeb7debd95017ae5a52a28071d6988f61b644/proto/waves/transaction.proto#L91

Reply:
Done